.gitconfig

```diff
@@ -8,7 +8,10 @@
 	ah = commit --amend -C HEAD
 	b = branch
 	bd = branch -D
-	c = commit -m
+	c = commit
+	ca = commit --amend
+	cl = clone
```

Comment:
Infrequent use case, but can't believe there wasn't a shorthand created for this when this was initially made